index.html

```diff
@@ -5,95 +5,131 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Wpgtk 4.0, color scheme generator</title>
+  <meta name="description" content="Wpg is a easy to use,
```

Comment:
`a easy` --> `an easy` (*This should also be changed in the repo near the title.*)